mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+private struct LeafText: View {
+    var body: some View {
+        Text("Evidence")
+            .subtitle()
+        Text("Who come first: the egg or the chicken?")
+            .title()
+        Text("The egg, sure!")
+            .body()
+        Text("1 like")
+            .label()
+        
+    }
+}
+
+#Preview {
+    LeafText()
+}
+
+//MARK: - TitleModifier
+public extension Text {
+    func title() -> some View {
+        @Environment(\.leafTheme)  var theme
```

Comment:
I'm not sure it's a good idea to use this inside a method like this. I think the `Environment` property wrapper relies on some `View` internal logic to work, maybe we should restrict it's usage to `View` properties and not use it inside methods.